lib/msal-node/src/cache/Storage.ts

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+    ICacheStorage,
+    Constants,
+    PersistentCacheKeys,
+    TemporaryCacheKeys,
+} from '@azure/msal-common';
+import { CacheOptions } from '../config/ClientConfiguration';
+import { CACHE } from './../utils/Constants';
+
+const fs = require('fs');
```

Comment:
You should be able to use `import`

Reply:
And I'd recommend using the `promise` API: https://nodejs.org/api/fs.html#fs_fs_promises_api

I think this should work:
```js
import { promises as fs } from 'fs'
```

Reply:
Sorry should have left a comment on this file (I did so on the previous PR that was accidently closed). Storage.ts not pertinent to us, I think it's here as a placeholder while Sameera works on the Cache.